website/docs/core-concepts/building-ui/designing-an-application/app-navigation.md

```diff
@@ -0,0 +1,87 @@
+# App Navigation
+
+We have introduced new navigation styles with better a default navigation - Top (Stacked), a variant for Top (Inline), and a collapsible Sidebar. You can now configure your app's navigation by opening the navigation settings tab inside the app settings pane and observe how your app with the selected navigation settings will look side by side as you change them.
+
+<figure>
+  <img src="/img/app-navigation-new-default.png" alt="The new default navigation - Top (Stacked)"/>
+  <figcaption align="center"><i>The new default navigation - Top (Stacked)</i></figcaption>
+</figure>
+
+## Navigation settings
+
+:::info
+Configuring the navigation is an application-level change.
```

Comment:
is application-level good enough to explain what it mean, maybe saying that `the scope of this change remains limited to the application where you set this` be better?